sub-packages/bionemo-llm/src/bionemo/llm/model/biobert/model.py

```diff
@@ -129,32 +129,12 @@ class BioBertOutput(BioBertOutputCore, total=False):
 
 # TODO make this a base class without the language head and pooler
 class MegatronBioBertModel(LanguageModule):
-    """Transformer language model.
-
-    Args:
-        config: transformer config
-        num_tokentypes: Set to 2 when args.bert_binary_head is True, and 0 otherwise. Defaults to 0.
-        transformer_layer_spec: Specifies module to use for transformer layers
-        vocab_size: vocabulary size
-        max_sequence_length: maximum size of sequence. This is used for positional embedding
-        pre_process: Include embedding layer (used with pipeline parallelism)
-        post_process: Include an output layer (used with pipeline parallelism)
-        parallel_output: Do not gather the outputs, keep them split across tensor parallel ranks
-        share_embeddings_and_output_weights: When True, input embeddings and output logit weights are shared.
-            Defaults to False.
-        position_embedding_type: Position embedding type. Options ["learned_absolute", "rope"].
-            Defaults is 'learned_absolute'.
-        rotary_percent: Percent of rotary dimension to use for rotary position embeddings.
-            Defaults to 1.0 (100%). Ignored unless position_embedding_type is 'rope'.
-    """
+    """Transformer language model."""
 
-    def __init__(  # noqa: D107
+    def __init__(
         self,
-        config: TransformerConfig,
+        config: "BioBertConfig",
         num_tokentypes: int,
-        transformer_layer_spec: ModuleSpec,
-        vocab_size: int,
-        max_sequence_length: int,
```

Comment:
Nearly all these arguments that we're passing around are _also_ in the `config` object, which we've mis-typed as a vanilla `TransformerConfig`